src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/custom.py

```diff
@@ -51,3 +51,24 @@ def list_vm(client, resource_group, lab_name, order_by=None, top=None,
 
     return client.list(resource_group, lab_name,
                        expand=expand, filter=filters, top=top, order_by=order_by)
+
+
+def get_formula(client, resource_group, lab_name, name, export_artifacts=None):
+    """ Command to show formula from the Azure DevTest Lab """
+    formula = client.get_resource(resource_group_name=resource_group, lab_name=lab_name, name=name)
+    if export_artifacts:
+        return _export_artifacts(formula=formula)
+
+    return formula
+
+
+def _export_artifacts(formula):
```

Comment:
Make sure to document the size effects: because properties are deleted from each item of the `artifacts` list, which is __not__ a copy of `formula.formula_content.artifacts`, the original sources are changed as well. This is not a problem in current usage pattern because the command returns the formula and finishes immediately. But it will construct issues when this method is reused in other settings when the `formula.formula content.artifacts` is reused.